src/panels/logbook/ha-logbook.ts

```diff
@@ -67,6 +84,10 @@ class HaLogbook extends LitElement {
     }
     const previous = this.entries[index - 1];
     const state = item.entity_id ? this.hass.states[item.entity_id] : undefined;
+    const item_username =
+      item.context_user_id && item.context_user_id in this._userid_to_name
+        ? this._userid_to_name[item.context_user_id]
+        : undefined;
```

Comment:
```suggestion
    const item_username = this._userid_to_name[item.context_user_id];
```

Reply:
I did `item.context_user_id && this._userid_to_name[item.context_user_id];` since `context_user_id` can be undefined and was giving a linter error

Reply:
ok :-)